test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -598,11 +598,17 @@ UCS_TEST_P(test_ucp_sockaddr, concurrent_disconnect_bidi) {
 }
 
 UCS_TEST_P(test_ucp_sockaddr, listen_inaddr_any) {
+    /* save testing address */
+    ucs::sock_addr_storage test_addr(m_test_addr);
     m_test_addr.reset_to_any();
 
     UCS_TEST_MESSAGE << "Testing " << m_test_addr.to_str();
 
     start_listener(cb_type());
+    /* get actual listening port */
+    test_addr.set_port(m_test_addr.get_port());
```

Comment:
why is this line needed? because the port could change if busy? pls expand comment..

Reply:
all the UCP tests start with port 0, so UCP will get random port, when listener is started test queries it to get actual port number. the listener creation can fail with BUSY only if different UCT CMs can't share the same port. Lets say, rdma-listener selected the port where tcp-listener can't be started.

I think these lines already explain what you requested:
https://github.com/openucx/ucx/blob/cefa5e20cb9810d8fccf3d117957b336c3b06178/test/gtest/ucp/test_ucp_sockaddr.cc#L154
https://github.com/openucx/ucx/blob/cefa5e20cb9810d8fccf3d117957b336c3b06178/test/gtest/ucp/test_ucp_sockaddr.cc#L187

Reply:
i mean, pls replace `/* get actual listening port */`
with smth like `/* initial port could be busy, so get the actual port which was selected */`

Reply:
but it's incorrect, initial port of `m_test_addr` is 0. this value is stored in begin of the test.
would "get the actual port which was selected by listener" be clear?

Reply:
yes, ok

Reply:
force-pushed since change is small